composables/plan.ts

```diff
@@ -53,3 +53,12 @@ export const useSubscriptionIdParam = () => {
   });
   return { subscriptionId };
 };
+
+export const usePaymentMethodParam = () => {
+  const route = useRoute();
+  const paymentMethodId = computed(() => {
+    const { m } = get(route).query;
+    return typeof m === 'string' ? Number(m) : -1;
```

Comment:
m is the payment duration in months right 1/3/6/12 we should change the naming to properly reflect this.

Reply:
okay, will do